.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -220,3 +249,70 @@ jobs:
             }
         env:
           SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN_AIRBYTE_TEAM }}
+
+
+  publish-docker-image:
+    timeout-minutes: 240
+    needs:
+      - start-publish-docker-image-runner-0
+      - publish-cdk
+    runs-on: runner-pool-${{ github.run_id }}
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+        with:
+          repository: airbytehq/airbyte
+          ref: master
+      - name: Install Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: "zulu"
+          java-version: "17"
+      - name: Install Python
```

Comment:
Is Python required? I didn't see where it was needed in the subsequent steps.

Reply:
That's a good question. `manage.sh` uses command like `./gradlew --no-daemon :airbyte-cdk:python:build -x test` (see https://github.com/airbytehq/airbyte/actions/runs/4296899471/jobs/7489230541#step:7:558) which I assumed uses Python. That being said, I haven't confirmed so if you think it shouldn't, let me know and I'll test the workflow without it

Reply:
Ah yep looks like you're right, I just overlooked that.